### PR TITLE
Editorial: recipients are not a conformance class

### PR DESCRIPTION
**Pull request by marcoscaceres**
Addresses https://github.com/w3ctag/design-reviews/issues/529

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/pull/66.html" title="Last updated on Mar 23, 2021, 6:58 AM UTC (9e8c701)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/66/6256b91...9e8c701.html" title="Last updated on Mar 23, 2021, 6:58 AM UTC (9e8c701)">Diff</a>

### CODE REVIEW

**Review by jyasskin — COMMENTED** [review]
These aren't strong preferences, just things I noticed after this came through my inbox.

File: index.html [hunk 1]
```diff
@@ -284,33 +284,43 @@ <h3>
...
+          requirements, developers need to read this section carefully and do
+          their best to adhere to the recommendations below. Developers should
+          also be aware that privacy laws in their jurisdictions can also
+          govern the usage and access to location data.
+        </p>
         <p>
-          Recipients MUST only request location information when necessary.
-          Recipients MUST only use the location information for the task for
-          which it was provided to them. Recipients MUST dispose of location
+          Recipients must only request location information when necessary.
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply index.html:302]: Yeah, I agree. They are good words tho :(  Maybe go with "ought to"?

[the remark this answers is not in the document]
Comment (jyasskin) [reply index.html]: Indeed, the https://tools.ietf.org/html/rfc6919#section-4 keywords are fine for non-normative sections. :)

**Review by jyasskin — APPROVED** [review]

**Comment by marcoscaceres** [comment]
heh, I didn't know about RFC6919. That's hilarious!

**Pull request merged**